### PR TITLE
~>> -> ~>

### PR DESCRIPTION
**Pull request by harrisi**
:)

### CODE REVIEW

**Comment by harrisi** [comment]
The credo issue is from using the latest circle ci orb, which uses Elixir 1.17. Credo 1.7.7 has a fix, but it might make sense for the orb to use a different Elixir version? Not sure what y'all think.

As a side note, the minimum Elixir version for this and Bundlex may want to bump to Elixir 1.13, since the `req` version used by Bundlex expects Elixir 1.13.

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ defmodule MyApp.Mixfile do
 
   defp deps() do
     [
-      {:unifex, "~>> 1.2.0"}
+      {:unifex, "~> 1.2.0"}
```
Comment (mat-hek) [inline README.md:42]: It should actually be

```suggestion
      {:unifex, "~> 1.2"}
```

;)

**Comment by harrisi** [comment]
Would you like me to update mix.lock in this PR or open another?

**Comment by harrisi** [comment]
I opened a separate PR for now. #115

**Comment by mat-hek** [comment]
Oh, it would have been enough to do that here ;) I see that some linting is still not passing, let's just merge this and I'll figure it out

**Pull request merged**